internal/provider/provider.go

```diff
@@ -672,6 +672,7 @@ func Provider() *schema.Provider {
 			"aws_msk_configuration": kafka.DataSourceConfiguration(),
 			"aws_msk_kafka_version": kafka.DataSourceVersion(),
 
+			"aws_mskconnect_connector":            kafkaconnect.DataSourceConnector(),
```

Comment:
```suggestion
			"aws_kafkaconnect_connector":            kafkaconnect.DataSourceConnector(),
```

Since both the [CLI v2](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/kafkaconnect/index.html) and [AWS SDK for Go v2](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/kafkaconnect) use `kafkaconnect`, let's also use `kafkaconnect` everywhere: service package (as you already have it), resource naming, label, connection (_i.e._, KafkaConnectConn), etc.

Reply:
Previous resources in this service have already been named `aws_mskconnect_thing`.